SonarRunner.Shim/Resources.resx

```diff
@@ -118,7 +118,7 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="MSG_CallingSonarRunner" xml:space="preserve">
-    <value>Calling the sonar-runner...</value>
+    <value>Calling the sonar-scanner...</value>
```

Comment:
*sonar-runner* is the name of the executable we are calling. If we're changing this to the product name then it should be "SonarQube Scanner" (multiple other occurrences of this).
@dbolkensteyn what do you think?